Tools/MiniBrowser/win/WebKitBrowserWindow.cpp

```diff
@@ -510,7 +510,21 @@ void WebKitBrowserWindow::decidePolicyForNavigationResponse(WKPageRef page, WKNa
         text << L"URL: " << createString(adoptWK(WKURLResponseCopyURL(response.get())).get());
         MessageBox(thisWindow.m_hMainWnd, text.str().c_str(), L"No Content", MB_OK | MB_ICONWARNING);
     }
-    WKFramePolicyListenerUse(listener);
+
+    if (WKNavigationResponseCanShowMIMEType(navigationResponse))
+        WKFramePolicyListenerUse(listener);
+    else {
+        std::wstringstream text;
+        text << L"Do you want to save this file ?" << std::endl;
```

Comment:
I think no space is needed before a question mark. " ?" → "?"

Reply:
Fixed